eyes_selenium/applitools/selenium/positioning.py

```diff
@@ -74,7 +74,7 @@ def get_entire_size(self):
             raise WebDriverException("Failed to extract entire size! \n{}".format(e))
         return RectangleSize(width=width, height=height)
 
-    def _add_data_attribute_to_element(self):
+    def add_data_attribute_to_element(self):
```

Comment:
Probably it's better to keep it private because the doc generation team would complain